configuration/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@azure-iot/configuration",
+  "version": "0.0.1",
```

Comment:
you might want to publish this as rc first. Take a look at other common libraries package.json 
And you can also test your library as an npm package by importing it to dm with "npm link" before publishing: https://docs.npmjs.com/cli/link